bin/lib/check_reqs.js

```diff
@@ -70,18 +72,23 @@ module.exports.get_target = function() {
 
 // Returns a promise. Called only by build and clean commands.
 module.exports.check_ant = function() {
-    return tryCommand('ant -version', 'Failed to run "ant -version", make sure you have ant installed and added to your PATH.');
+    return tryCommand('ant -version', 'Failed to run "ant -version", make sure you have ant installed and added to your PATH.')
+    .then(function (output) {
+        // Parse Ant version from command output
+        return /version ((?:\d+\.)+(?:\d+))/i.exec(output)[1];
+    });
 };
 
 // Returns a promise. Called only by build and clean commands.
 module.exports.check_gradle = function() {
     var sdkDir = process.env['ANDROID_HOME'];
-    var wrapperDir = path.join(sdkDir, 'tools', 'templates', 'gradle', 'wrapper');
-    if (!fs.existsSync(wrapperDir)) {
-        return Q.reject(new Error('Could not find gradle wrapper within android sdk. Might need to update your Android SDK.\n' +
-            'Looked here: ' + wrapperDir));
-    }
-    return Q.when();
+    var wrapper = path.join(sdkDir, 'tools', 'templates', 'gradle', 'wrapper', 'gradlew');
```

Comment:
If `ANDROID_HOME` is not defined, the error you will get here is:

    Gradle: not installed
    [TypeError: Arguments to path.join must be strings]

(since you end up passing null to `path.join()`)

While there is a previous check for `ANDROID_HOME` (as part of the Android SDK check), it would be nice to make this error more meaningful.